java/compaction/compaction-core/src/main/java/sleeper/compaction/job/commit/CompactionJobCommitterOrSendToLambda.java

```diff
@@ -58,7 +58,7 @@ public CompactionJobCommitterOrSendToLambda(
 
     public void commit(CompactionJob job, CompactionJobFinishedEvent.Builder finishedBuilder) throws StateStoreException {
         boolean commitAsync = tablePropertiesProvider.getById(job.getTableId()).getBoolean(COMPACTION_JOB_COMMIT_ASYNC);
-        CompactionJobFinishedEvent finishedEvent = finishedBuilder.committedBySeparateUpdate(true).build();
+        CompactionJobFinishedEvent finishedEvent = finishedBuilder.build();
```

Comment:
We could just pass the CompactionJobFinishedEvent into the method now rather than the builder?